app/javascript/components/patient/assessment/AssessmentTable.js

```diff
@@ -111,11 +112,63 @@ class AssessmentTable extends React.Component {
               const symptomColData = this.getSymptomCols(response.data.symptoms);
               updatedColData = state.table.colData.concat(symptomColData);
             }
+
+            // The following section creates aria-label strings for the Assessments table. There are four possible scenarios:
```

Comment:
is it worth breaking this out into it's own function? it's a bit complicated

Reply:
Yeah I can do that